draft-private-access-tokens.md

```diff
@@ -324,7 +387,7 @@ For example, an Issuer URL might be https://issuer.net/access-token-request.
 Mediators advertise a URL for proxying protocol messages to Issuers. For example,
 a Mediator URL might be https://mediator.net/relay-access-token-request.
 
-# Protocol
+# Protocol {#protocol}
```

Comment:
Markdown automagically generates this, so this may not be needed?

Reply:
Didn't realize that... but might be easier for consistency with other sections

Reply:
Yeah, whatever works :) Feel free to ignore this comment.